content/blog/2023/best-practices-deploying-pipelines-with-hpc-workload-managers.md

```diff
@@ -327,8 +321,7 @@ $ cat submit_pipeline.sh
 #BSUB -e err.%J
 #BSUB -J headjob
 #BSUB -R "rusage[mem=16GB]"
-export NFX_OPTS="-Xms=512m -Xmx=8g"
```

Comment:
Removed as this isn't explained until the following section. Alternatively could reorder the sections, or just leave it here without explanation is probably fine too.